### ISSUE TITLE
Share playlist (e.g. as an m3u file)

### ISSUE THREAD

**Issue by ericsfraga** [gateship-one/odyssey#241]
# Feature request

I have created a playlist in Odyssey. I would like to share this playlist (e.g. as an m3u file) with another app or person. It may already be possible if I could find where playlists are stored in the file system (if in a "public" area).

**Comment by MaxsimLionel** [comment]
Yes, I second this. Where are playlists stored? And are they m3u files?

**Comment by gnome17** [comment]
Hi,

we started by using the playlist feature of android itself but since the support was dropped we switched to an internal implementation. They can only be accessed by odyssey itself and are not m3u files.

We never thought about a share function of playlists so this feature is not yet available.

**Comment by ericsfraga** [comment]
Okay, thank you. No worries. Android is a moving target... with little regard for anybody really.
